### DISCUSSION TITLE
非常好项目，使我coding旋转

### DISCUSSION THREAD

**Discussion by c4yg70** [ygqygq2/turbo-file-header#2] [General]
在Coding-CI里面添加了云原生开发调用本扩展的支持~

**Comment by ygqygq2** [comment]
> 在Coding-CI里面添加了云原生开发调用本扩展的支持~

终于有人认可，不容易呀 😅

**Reply by c4yg70** [reply]
我觉得功能控制得挺好的，各方面都比koro要清爽一些~（尤其启动时间orz）
